backend/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "backend"
+version = "0.1.0"
+default-run = "backend"
+edition = "2021"
+description = "Bob Management GUI: Backend"
+authors = ["Romanov Simeon ArchArcheoss@proton.me", "Artemev Ilia", "Riadinskii Kirill"]
+license = "Unlicense"
+repository = "https://github.com/qoollo/bob-management"
+publish = false
+keywords = [ "BOB", "Management", "GUI" ]
+readme = "./README.md"
+
+[dependencies]
+# Axum related
+axum = "0.6"
+axum-macros = "0.3"
+axum-login = "0.5"
+axum-sessions = "0.5"
+tower = "0.4"
+tower-http = { version = "0.4", features = ["cors"] }
+
+# Logging
+tracing = "0.1"
+tracing-subscriber = "0.3"
+
+# Error Handling
+color-eyre = "0.6"
+thiserror = "1.0"
+
+# General
+tokio = { version = "1.0", features = ["full"] }
```

Comment:
Latest version of tokio is `1.32`

Reply:
In general it is better to add the minimal required set of features (`full` is not good)